src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -295,7 +294,7 @@ For many apps, user sign-up and sign-in with a username/password is all that is
 <Accordion title='Review the sign-up, sign-in, and sign-out workflow' headingLevel='3' eyebrow='Learn more'>
 
 Below is a high-level overview of the workflows for sign-up, sign-in, and sign-out with Amplify authentication:
-
+ 
 **Sign-up:** The username/password set up will ask your user for a username (or email) and a password. This should also include any custom attributes you include in your sign-up from such as address, birthdate, phone, etc. When your end-user submits this information, it is then passed through the `Auth.signUp()` method to your Amazon Cognito User Pool, where a user is created. By default, the user will receive a confirmation message via email or SMS, depending on what you configured, but you have the option to configure auto confirmation. They must then enter the confirmation code to complete the sign-up process. Once the user is verified, their user profile is officially confirmed and is active in the Cognito User Pool. The profile contains the username, password, email, and any other custom attributes collected during sign-up. This user profile can then be used to sign-in and access your application.
```

Comment:
remove "custom attributes". It is just "attributes.

Reply:
The profile contains the username, password, and any other custom attributes collected during sign-up.